src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateHoldingEventHandler.java

```diff
@@ -58,6 +66,8 @@ public class CreateHoldingEventHandler implements EventHandler {
   private static final String MAPPING_METADATA_NOT_FOUND_MSG = "MappingMetadata snapshot was not found by jobExecutionId '%s'. RecordId: '%s', chunkId: '%s' ";
   static final String ACTION_HAS_NO_MAPPING_MSG = "Action profile to create a Holding entity requires a mapping profile";
   private static final String FOLIO_SOURCE_ID = "f32d531e-df79-46b3-8932-cdd35f7a2264";
+  private static final String ERRORS = "ERRORS";
+  private static final String BLANK = "";
```

Comment:
Could we use https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#EMPTY constant?